scripts/release.py

```diff
@@ -131,25 +124,15 @@ def get_changelog_release_notes(release_notes_url: str, version: str) -> str:
     if match:
         return str(match.group(1)).strip()
     else:
-        print(
-            f"WARNING: Failed to parse changelog release notes. Manually copy this version's notes from the CHANGELOG.md file to {release_notes_url}.",
```

Comment:
Haven't tripped on this error handler yet, but I think the URL in it was wrong. It's a _compare_ link like https://github.com/john-kurkowski/tldextract/compare/5.1.1...5.1.2, not a create-release link. I moved this error print to after we have the release link, with a hardcoded URL.

Reply:
Yeah, good point. I think I garbled this warning when I added the `release_notes_url`. Your new placement is correct: it should be directing the user to the url of the newly created draft release to manually add the notes. Thanks for catching this!